.github/workflows/pytest-min-versions.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
           ${{ runner.os }}-${{ matrix.python-version }}-pip-
     - name: generate min_requirements.txt
       run: |
-        pip install requirements-builder
+        pip install git+https://github.com/jenshnielsen/requirements-builder.git
```

Comment:
since this is used only for OUR CI it's ok to use the fork. hope the upstream will be updated soon :)